src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java

```diff
@@ -0,0 +1,188 @@
+package hlf.java.rest.client.service.impl;
+
+import static hlf.java.rest.client.model.ChaincodeOperationsType.approve;
+import static java.util.Objects.isNull;
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+import hlf.java.rest.client.exception.ErrorCode;
+import hlf.java.rest.client.exception.ServiceException;
+import hlf.java.rest.client.model.ChaincodeOperations;
+import hlf.java.rest.client.model.ChaincodeOperationsType;
+import hlf.java.rest.client.service.ChaincodeOperationsService;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Collection;
+import java.util.concurrent.CompletableFuture;
+import lombok.extern.slf4j.Slf4j;
+import org.hyperledger.fabric.gateway.Gateway;
+import org.hyperledger.fabric.gateway.Network;
+import org.hyperledger.fabric.gateway.impl.identity.X509IdentityProvider;
+import org.hyperledger.fabric.sdk.BlockEvent;
+import org.hyperledger.fabric.sdk.Channel;
+import org.hyperledger.fabric.sdk.HFClient;
+import org.hyperledger.fabric.sdk.LifecycleApproveChaincodeDefinitionForMyOrgProposalResponse;
+import org.hyperledger.fabric.sdk.LifecycleApproveChaincodeDefinitionForMyOrgRequest;
+import org.hyperledger.fabric.sdk.LifecycleCommitChaincodeDefinitionProposalResponse;
+import org.hyperledger.fabric.sdk.LifecycleCommitChaincodeDefinitionRequest;
+import org.hyperledger.fabric.sdk.Peer;
+import org.hyperledger.fabric.sdk.exception.CryptoException;
+import org.hyperledger.fabric.sdk.exception.InvalidArgumentException;
+import org.hyperledger.fabric.sdk.exception.ProposalException;
+import org.hyperledger.fabric.sdk.security.CryptoSuite;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class ChaincodeOperationsServiceImpl implements ChaincodeOperationsService {
+
+  @Autowired private Gateway gateway;
+
+  @Override
+  public String performChaincodeOperation(
+      String networkName,
+      ChaincodeOperations chaincodeOperationsModel,
+      ChaincodeOperationsType operationsType) {
+
+    validateChaincodeOperationsInput(chaincodeOperationsModel, operationsType);
+    HFClient hfClient = HFClient.createNewInstance();
+    try {
+      hfClient.setCryptoSuite(CryptoSuite.Factory.getCryptoSuite());
+      X509IdentityProvider.INSTANCE.setUserContext(
+          hfClient, gateway.getIdentity(), "hlf-rest-client");
```

Comment:
@jopuneet Can u change it to hlf-connector?

Reply:
@anandbanik Done. thank you

Reply:
@jopuneet Thanks!